src/index.js

```diff
@@ -12,24 +12,27 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin')
  * -------------------------------------------------------------------------- */
 
 module.exports = (neutrino, options = {}) => {
-  // We want to start from a clean slate. This removes any existing "style"
-  // rule that may exist (present if using the default neutrino-preset-web).
-  neutrino.config.module.rules.delete('style')
-
+  const { loaderOptions = {}, pluginOptions = {} } = options
   const styleRule = neutrino.config.module
     .rule('style')
-    .test(options.test || /\.css$/)
+    .test(loaderOptions.test || /\.css$/)
 
   const loaders = ExtractTextPlugin.extract({
-    fallback: options.fallback || 'style-loader',
-    use: options.use || 'css-loader'
+    fallback: loaderOptions.fallback || 'style-loader',
+    use: loaderOptions.use || 'css-loader'
   })
 
+  // We want to start from a clean slate. This removes any existing "style"
+  // rule that may exist (present if using the default neutrino-preset-web).
+  neutrino.config.module.rules.delete('style')
+
+  pluginOptions.filename = pluginOptions.filename || '[name].css'
+
   loaders.forEach(({ loader, options }) => styleRule.use(loader)
     .loader(loader)
     .options(options)
   )
 
   neutrino.config.plugin('extract')
-    .use(ExtractTextPlugin, [options.filename || '[name].css'])
+    .use(ExtractTextPlugin, [pluginOptions])
```

Comment:
How do you feel about using object assign here? Would let us remove the line above.

```js
[Object.assign({ filename: '[name].css' }, pluginOptions)]
```